torchci/pages/kpis.tsx

```diff
@@ -72,15 +72,37 @@ export default function Kpis() {
 
       <Grid item xs={12} lg={6} height={ROW_HEIGHT}>
         <TimeSeriesPanel
-          title={"Time to Red Signal - (Weekly)"}
+          title={"Time to Red Signal - (Weekly, pull workflow)"}
           queryName={"ttrs_percentiles"}
-          queryCollection={"pytorch_dev_infra_kpis"}
-          queryParams={[...timeParams]}
+          queryParams={{
+            ...clickhouseTimeParams,
+            one_bucket: false,
+            percentile_to_get: 0,
+            workflow: "pull",
+          }}
           granularity={"week"}
           timeFieldName={"bucket"}
           yAxisFieldName={"ttrs_mins"}
           yAxisRenderer={(duration) => duration}
-          groupByFieldName={"percentile"}
+          useClickHouse={true}
+          groupByFieldName="percentile"
+          // Format data so that we can display all percentiles in the same
+          // chart. Goes from 1 row per timestamp with all percentiles in the
+          // row to 4 rows per timestamp with one percentile in each row.
+          dataReader={(data) => {
```

Comment:
Why do you think the old way doesn't work? 

Looking at the old and new versions of the kpis page the charts look exactly the same to me

<img width="2540" alt="image" src="https://github.com/user-attachments/assets/e6f13b66-8953-4886-b772-fbb81aba0d54">

Reply:
The old one was using Rockset.  This PR switches it to use ClickHouse